src/main/java/ch/njol/skript/lang/UnparsedLiteral.java

```diff
@@ -108,248 +106,120 @@ public <R> Literal<? extends R> getConvertedExpression(final ParseContext contex
 		} finally {
 			log.stop();
 		}
-		
```

Comment:
I hope nobody needed this 😬

Reply:
well nobody needed it for the last 12 years so i think it's ok :)

Reply:
Are we talking about this blank line? 
I believe SkBee relies on it... I think it should be added back. (/s)

Reply:
lol no it was meant to be attached to a long section of commented-out code that I removed.